llvm/lib/Transforms/Vectorize/VPlan.h

```diff
@@ -2285,10 +2285,16 @@ class VPWidenPHIRecipe : public VPSingleDefRecipe {
   /// List of incoming blocks. Only used in the VPlan native path.
   SmallVector<VPBasicBlock *, 2> IncomingBlocks;
 
+  /// Name to use for the generated IR instruction for the widened IV.
+  std::string Name;
```

Comment:
Maybe use SmallString or Twine?

Reply:
std::string seems to be what everything else in VPlan.h uses, and it doesn't look like Twine can be stored